modules/camera/camera_macos.mm

```diff
@@ -76,7 +68,11 @@ - (id)initForFeed:(Ref<CameraFeed>)p_feed andDevice:(AVCaptureDevice *)p_device
 		if (!output) {
 			print_line("Couldn't get output device for camera");
 		} else {
-			NSDictionary *settings = @{ (NSString *)kCVPixelBufferPixelFormatTypeKey : @(kCVPixelFormatType_420YpCbCr8BiPlanarFullRange) };
+			NSDictionary *settings = @{
+				(NSString *)kCVPixelBufferPixelFormatTypeKey : @(kCVPixelFormatType_420YpCbCr8BiPlanarFullRange),
+				@"Width" : @1280,
+				@"Height" : @720,
```

Comment:
Note that recent MacBooks have 1080p webcams, so this should be configurable somehow.

Reply:
In my opinion there are two approaches to solve this issue:

1. Define a feed for each combination of supported dimension and data format and let the user choose to most appropriate feed.
2. Define a feed per camera and make the preferred dimension and data format configurable.

Who is responsible for achitectural changes to the Camera feature? He/she should make this call so that conflicting PRs can be resolved.